test/index.js

```diff
@@ -361,13 +360,118 @@ lab.experiment('Normal setup', function (done) {
             },
             config : {
                 plugins: {
-                    policies: [{object: 'invalidPlainObject'}]
+                    policies: [ {object: 'invalidPlainObject'} ]
+                }
+            }
+        });
+
+        server.route({
+            method : 'GET',
+            path   : '/parallel-ok',
+            handler: function (request, reply) {
+
+                reply({
+                    handler: 'handler'
+                });
+            },
+            config : {
+                plugins: {
+                    policies: [ MrHorse.parallel('postHandler', 'postHandlerAnother') ]
+                }
+            }
+        });
+
+        server.route({
+            method : 'GET',
+            path   : '/parallel-fails',
+            handler: function (request, reply) {
+
+                reply({
+                    handler: 'handler'
+                });
+            },
+            config : {
+                plugins: {
+                    policies: [ MrHorse.parallel('passes', 'fails') ]
```

Comment:
It looks like we don't have a test with two failing policies.  If that's correct, it seems like it would be valuable to have one and verify that the first failing test is the one that gets reported back.